eos-tech-support/eos-download-image

```diff
@@ -298,8 +298,16 @@ class EosDownloadImage(object):
     def fetch_manifest(self):
         manifest_url = '{base}/releases-{product}-3.json'.format(
             base=self.base, product=self.args.product)
-        manifest = self.session.get(manifest_url).json()
+        response = self.session.get(manifest_url)
 
+        try:
+            response.raise_for_status()
+        except requests.exceptions.HTTPError as e:
+            log("Couldn't fetch manifest for product", self.args.product)
+            log(e)
+            sys.exit(1)
+
+        manifest = response.json()
```

Comment:
Updated to add this line, without which, of course, nothing works. Apparently I tested the error path, but not the happy path.